solutions/3653/01.py

```
@@ -0,0 +1,17 @@
+class Solution:
+    def xorAfterQueries(self, nums: List[int], queries: List[List[int]]) -> int:
```

Ruff (0.14.8)

Undefined name `List`

(F821)